### PR TITLE
downloads + swf2exe

### PR DESCRIPTION
**Pull request by dignitysr**

### CODE REVIEW

File: docs/games/sm63/sm63.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # Super Mario 63
 
-Super Mario 63, or SM63 for short, is a fan-game by [Runouw](https://runouw.com) based on [Super Mario 64](https://wikipedia.org/wiki/Super_Mario_64), [Super Mario Sunshine](https://wikipedia.org/wiki/Super_Mario_Sunshine), and [Super Mario Galaxy](https://wikipedia.org/wiki/Super_Mario_Galaxy).
\ No newline at end of file
+Super Mario 63, or SM63 for short, is a fan-game by [Runouw](https://runouw.com) based on [Super Mario 64](https://wikipedia.org/wiki/Super_Mario_64), [Super Mario Sunshine](https://wikipedia.org/wiki/Super_Mario_Sunshine), and [Super Mario Galaxy](https://wikipedia.org/wiki/Super_Mario_Galaxy).
```
Comment (GTcreyon) [inline docs/games/sm63/sm63.md:3]: Why is this line showing up as changed? What changed?

File: docs/tools/swf2exe.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Swf2Exe is a tool that can be used to convert SWF files into EXE files for easier access. 
+
+## Download
+
+- [Download the tool here](https://cdn.discordapp.com/attachments/711545571130998846/795404822882418738/Swf2eXe_Latest.zip)
+
+## Usage
+
+To effectively use the tool, extract the ZIP file to any folder of your choice and then open the sWf2eXe executable. Press the "Clear Input" button to removed the placeholder SWF then press the "Add a File" button and select the desired SWF.
```
Comment (GTcreyon) [inline docs/tools/swf2exe.md]: - Remove "effectively"
- Fix capitalisation on sWf2eXe
- "removed" -> "remove"

File: docs/tools/swf2exe.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+Once you have added your SWF file, simply press convert, and the tool will convert the SWF to an EXE and place it in the Output folder that can be found in the same directory.
```
Comment (GTcreyon) [inline docs/tools/swf2exe.md]: Remove "simply"

File: mkdocs.yml [hunk 4]
```diff
@@ -17,4 +18,6 @@ theme:
   locale: en
   highlightjs: true
   hljs_languages:
-    - actionscript
\ No newline at end of file
+    - actionscript
```
Comment (GTcreyon) [inline mkdocs.yml:21]: Preferably add a whitespace line here.

**Review by GTcreyon — CHANGES_REQUESTED** [review]

File: docs/games/sm63/sm63.md (hunk 1 above)
Comment (dignitysr) [reply docs/games/sm63/sm63.md:3]: good question

Comment (GTcreyon) [reply docs/games/sm63/sm63.md:3]: Did you just accidentally add a space at the end? Is it some encoding thing? Either way, it's probably not important.

**Review by GTcreyon — APPROVED** [review]

**Pull request merged**